### PR TITLE
Feature/handlebars html generation

### PR DESCRIPTION
**Pull request by le0m**
This PR adds functions to generate the content of `index.html` files for all kinds of AR.js applications.

Currently provided applications:
- barcode with 3d model
- pattern with 3d model
- pattern with image
- pattern with audio
- pattern with video

### CODE REVIEW

**Comment by nicolocarpignoli** [comment]
@le0m please review my changes, added some fixes on handlebars template (some of them were using old repository build of AR.js, and aligned some property here and there).

File: src/modules/location/templates/location.3d.handlebars [hunk 1]
```diff
@@ -4,23 +4,23 @@
...
             vr-mode-ui="enabled: false"
-            arjs="sourceType: webcam; sourceWidth:1280; sourceHeight:960; displayWidth: 1280; displayHeight: 960; debugUIEnabled: false;"
+            arjs="sourceType: webcam; debugUIEnabled: false;"
             embedded
         >
             <a-entity
                 gltf-model="{{assetSrc}}"
-                rotation="0 180 0"
-                scale="1 1 1"
+                look-at="[gps-camera]"
```
Comment (le0m) [inline src/modules/location/templates/location.3d.handlebars]: Is this a parameter that we should handle with handlebar? Or it's a normal string?

Comment (nicolocarpignoli) [reply src/modules/location/templates/location.3d.handlebars]: I added the import of that component, 'look-at', and the default use of it. It is intended to be use as default for location based, in my opinion. We should keep it, at least for now :)

Comment (nicolocarpignoli) [reply src/modules/location/templates/location.3d.handlebars]: So to answer you, I would use it as I added it, hardcoded.

**Comment by nicolocarpignoli** [comment]
@le0m if it's okay for you, We can merge this one

**Pull request merged**